package.json

```diff
@@ -69,10 +78,9 @@
   },
   "dependencies": {
     "dom-helpers": "^3.3.1",
-    "global": "^4.3.2",
-    "invariant": "^2.2.4",
-    "react": "^16.3.0",
-    "react-reconciler": "0.10.0",
-    "warning": "^3.0.0"
+    "fbjs": "^0.8.17"
```

Comment:
i really don't want a dep on fbjs if we can help it, its more of an internal FB project, and REact itself is moving away from including it

Reply:
Got it. It helped us avoid duplicate declarations for warning and other modules.

Reply:
yeah i wonder if it's worth making out own one for that? I have a package react-tackle-box i dump random common stuff into, if it's helpful i could put stuff there